WebAPI.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,89 @@
+using System.Text.Json.Serialization;
+using RestSharp;
+using WebAPI.Infrastructure;
+using WebAPI.Infrastructure.RequestsResponses;
+
+namespace WebAPI.Tests;
+
+public class Tests
+{
+    private RestApiClient _restClient;
+    private int _id;
+
+    [SetUp]
+    public async Task Setup()
+    {
+        _restClient = new RestApiClient();
+        await _restClient.Authenticate("admin", "password123");
```

Comment:
Let's not hardcode credentials in the tests level. Please create class UserStorage or smth like this and put credentials there